---
title: "Comprehensive detection and dissection of interlineage recombination events in the SARS-CoV-2 pandemic"
collection: publications
category: manuscripts
permalink: /publication/2024a
excerpt: 'A computational method to identify interlineage SARS-CoV-2 recombinants.'
date: 2024-09-02
venue: 'Virus Evolution'
paperurl: 'https://academic.oup.com/ve/advance-article/doi/10.1093/ve/veae074/7749618'
citation: 'Li, Jia-Ying, et al. "Comprehensive detection and dissection of interlineage recombination events in the SARS-CoV-2 pandemic." Virus Evolution (2024): veae074.'
---

# Abstract
The global prevalence of the XBB lineage presents a formidable challenge posed by the recombinant SARS-CoV-2 virus. The understanding of SARS-CoV-2â€™s recombination preference assumes utmost significance in predicting future recombinant variants and adequately preparing for subsequent pandemics. Thus, an urgent need arises to establish a comprehensive landscape concerning SARS-CoV-2 recombinants worldwide and elucidate their evolutionary mechanisms. However, the initial step, involving the detection of potential recombinants from a vast pool of over ten million sequences, presents a significant obstacle. In this study, we present CovRecomb, a lightweight methodology specifically designed to effectively identify and dissect interlineage SARS-CoV-2 recombinants. Leveraging CovRecomb, we successfully detected 135,567 putative recombinants across the entirety of 14.5 million accessed SARS-CoV-2 genomes. These putative recombinants could be classified into 1,451 distinct recombination events, of which 206 demonstrated transmission spanning multiple countries, continents, or globally. Hotspot regions were identified in six specific areas, with prominence observed in the latter halves of the N-terminal domain and receptor-binding domain within the *spike* (*S*) gene. Epidemiological investigations revealed extensive recombination events occurring among different SARS-CoV-2 (sub)lineages, independent of lineage prevalence frequencies.
